docs/pages/includes/preset-roles-table.mdx

```diff
@@ -0,0 +1,13 @@
+| Role | Description |
+| --- | --- |
+| `access`| Allows access to cluster resources. |
+| `auditor`| Allows reading cluster events, audit logs, and playing back session records. |
+| `device-admin`| Used to manage trusted devices.| 
+| `device-enroll`| Used to grant device enrollment powers to users.|
+| `editor` | Allows editing of cluster configuration settings. |
+| `group-access`| Allows access to all user groups.| 
+| `requester`| Enterprise-only role that allows a user to create Access Requests. |
+| `require-trusted-device`| Requires trusted device access to resources.|
+| `reviewer`| Enterprise-only role that allows review of Access Requests. |
+| `terraform-provider`|Allows the Teleport Terraform provider to configure all of its supported Teleport resources.|
```

Comment:
take it or leave it, but as a reader I think I'd rather have the roles I'm going to need day-0 listed near the top, and the rest grouped by purpose

```suggestion
| `access`| Allows access to cluster resources. |
| `editor` | Allows editing of cluster configuration settings. |
| `auditor`| Allows reading cluster events, audit logs, and playing back session records. |
| `requester`| Enterprise-only role that allows a user to create Access Requests. |
| `reviewer`| Enterprise-only role that allows review of Access Requests. |
| `group-access`| Allows access to all user groups. | 
| `device-admin`| Used to manage trusted devices. | 
| `device-enroll`| Used to grant device enrollment powers to users. |
| `require-trusted-device`| Requires trusted device access to resources. |
| `terraform-provider`| Allows the Teleport Terraform provider to configure all of its supported Teleport resources. |
```